src/portals/api.rs

```diff
@@ -126,15 +128,15 @@ pub enum AsPortalDestination {
 }
 
 /// Portal destination to be created
-#[derive(Clone, Default)]
+#[derive(Clone, Reflect, FromReflect, Default)]
```

Comment:
`CreatePortalDestination` and `AsPortalDestination` might not need it there ; I tried to add it because I wanted to register_type for `CreatePortal` too, but `Face`from wgpu is preventing me to do it, not sure how to fix it right now